content/en/docs/tasks/observability/logs/otel-provider/test.sh

```diff
@@ -20,11 +20,12 @@ set -u
 set -o pipefail
 
 source "tests/util/samples.sh"
+source "content/en/boilerplates/snips/start-otel-collector-service.sh"
 
 # @setup profile=demo
 
 # Start the otel sample
-startup_otel_sample
+bpsnip_start_otel_collector_service__1
```

Comment:
Maybe not absolutely necessary, but I would add a wait_for_deployment call.

```suggestion
bpsnip_start_otel_collector_service__1
_wait_for_deployment istio-system opentelemetry-collector
```